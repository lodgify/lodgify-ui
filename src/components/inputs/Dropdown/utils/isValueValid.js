/**
 * @param  {boolean|number|string} currentValue
 * @return {boolean}
 */
export const isValueValid = currentValue =>
  !!currentValue || currentValue === 0;
