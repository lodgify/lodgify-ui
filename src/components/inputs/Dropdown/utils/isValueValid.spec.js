import { isValueValid } from './isValueValid';

describe('`isValueValid`', () => {
  const testCases = [[undefined, undefined], [0, 0], ['a', 'b']];

  testCases.forEach(caseArguments => {
    it('should return the right value', () => {
      const actual = isValueValid(...caseArguments);

      expect(actual).toMatchSnapshot();
    });
  });
});
