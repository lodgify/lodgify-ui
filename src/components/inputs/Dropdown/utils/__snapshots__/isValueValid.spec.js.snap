// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`isValueValid\` should return the right value 1`] = `false`;

exports[`\`isValueValid\` should return the right value 2`] = `true`;

exports[`\`isValueValid\` should return the right value 3`] = `true`;
